docs/architecture/cloud-native/azure-development-stack.md

```diff
@@ -1,137 +1,143 @@
 ---
-title: The Azure Cloud Native Development Stack
-description: Architecting Cloud Native .NET Apps for Azure | The Azure Cloud Native Development Stack
-author: ardalis
-ms.date: 06/30/2019
+title: The Azure Cloud-Native Development Stack
```

Comment:
Microsoft's style asks for sentence case for headings and titles
https://docs.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings

```suggestion
title: The Azure cloud-native development stack
```